update dependencies

mstimberg opened the pull request:
Adds `lxml` dependency, and `typing` for old Python versions (in particular Python 2).﻿

mstimberg committed:
update dependencies

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

mstimberg committed:
Install the typing package unconditionally

The typing package is part of the stdlib for Python >=3.5, so installing the typing
package is not necessary on these versions. However, restricting its installation
to Python versions < 3.5 will prevent the conda package from being "noarch". Since
the stdlib takes precedence over installed packages, having an unnecessary typing
package should not hurt. This should be removed when support for Python < 3.5 is
no longer an issue.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

pgleeson commented:
Looks good. Thanks!
